test/test_peakmap.py

```diff
@@ -18,6 +18,8 @@
     ],
 )
 def test_peakmap_plot(featureMap_data, snapshot, kwargs):
+    if featureMap_data.empty:
+        pytest.skip("DataFrame is empty, skipping plot.")
```

Comment:
This line (and in functions after not need this as by definition this should always be present)